src/system_database.ts

```diff
@@ -37,7 +37,7 @@ export interface SystemDatabase {
   sleepms(workflowUUID: string, functionID: number, duration: number): Promise<void>;
 
   send<T>(workflowUUID: string, functionID: number, destinationUUID: string, message: T, topic?: string): Promise<void>;
-  recv<T>(workflowUUID: string, functionID: number, topic?: string, timeoutSeconds?: number): Promise<T | null>;
+  recv<T>(workflowUUID: string, functionID: number, topic?: string, timeoutSeconds?: number, timeoutDurable?: boolean): Promise<T | null>;
```

Comment:
I don't think we need this `timeoutDurable` flag. Because the old behavior (resetting the timer) is not ideal and more like a bug we want to fix.

Reply:
timeoutDurable  removed